services/object/schemas/v1/download-object-request.yml

```diff
@@ -0,0 +1,17 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Download Object Request"
+description: |
+  Download object request.
+type:               object
+properties:
+  acceptProtocols:
+    type: array
+    title: "Supported protocols"
```

Comment:
Let's get some additional documentation in here about what "protocol" means in this context (I think it's what I was calling "approach", right?  I agree that "approach" isn't a great word for it).

Rather than write several paragraphs in the schema, it might make sense to put this in a document under `ui/docs/reference` (e.g., "Download Protocols") and just refer to that document here.

Reply:
Sounds good. We can rename to download methods.